src/main/java/frc/robot/subsystems/swerve/module/SwerveModule.kt

```diff
@@ -51,8 +55,26 @@ class SwerveModule(
             150 / 7.0,
             2.0 measuredIn Inches
         )
+        RobotType.Mode.REAL -> ModuleIOTorqueCurrent(
+            driveID,
+            6.75,
```

Comment:
Can we get rid of the magic numbers, or at least comment what they mean, in this? Also line 65